### PR TITLE
Homework2: Anton Kochnevski

### PR DESCRIPTION
**Pull request by Antonkoch999**
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle)
- [x] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks
- [x] my PR made from the fork feature branch to maxim homework# branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore

### CODE REVIEW

File: src/homework2/task1.py [hunk 1]
```diff
@@ -14,10 +14,13 @@ def total_sum(m, n, s):
         'x rubles y kopecks'
     """
     # write your code here
-    return ''  # write return value here
+    m = m * 100
+    n = m + n
```
Comment (belovmd) [inline src/homework2/task1.py]: can be made in one line

File: src/homework2/task2.py [hunk 2]
```diff
@@ -14,10 +14,19 @@ def longest_word(str_):
     """
 
     # write your code here
-    return ''  # write return value here
+    import string
+    punct = string.punctuation + ' '
+    for el in punct:
```
Comment (belovmd) [inline src/homework2/task2.py]: O(n*m) solution. n - number of marks, m - length of the string. 
can be made with O(m)

**Review by belovmd — COMMENTED** [review]
Thank you for your work. 
Please address next paragraphs from review notes [Review Notes](https://github.com/belovmd/it-academy-summer/blob/master/Homework%20review/Homework2.md):
- 18
- 19

File: src/homework2/task1.py (hunk 1 above)
Comment (Antonkoch999) [reply src/homework2/task1.py]: done

File: src/homework2/task2.py (hunk 2 above)
Comment (Antonkoch999) [reply src/homework2/task2.py]: done

**Comment by Antonkoch999** [comment]
Ready to check!

**Review by belovmd — COMMENTED** [review]
9,5
(-0.5 for namings in task 7)

**Pull request closed without merging**